artifacts/fake-templates/android-sdk.yaml

```diff
@@ -0,0 +1,38 @@
+apiVersion: v1
+kind: Template
+metadata:
+  name: android-sdk
+  annotations:
+    openshift.io/display-name: Android SDK Installer!
+    description: "Installer to add Android SDK to the Build Farm (Mobile CI/CD). Before downloading the Android SDK Installer, you must agree to the terms and conditions of the Android Software Development Kit License Agreement, https://developer.android.com/studio/terms.html.\n\n"
+    iconClass: fa fa-android
+    tags: JENKINS,ci cd,mobile-service
+    template.openshift.io/long-description: Not sure if we could here _include_ the actual license, in a textbox/textarea.
+    template.openshift.io/provider-display-name: Feedhenry
+    template.openshift.io/documentation-url: https://github.com/aerogear/digger-jenkins
+    template.openshift.io/support-url: https://github.com/aerogear/digger-jenkins
+labels:
+  template: android-sdk
+objects:
+- apiVersion: v1
+  kind: Service
+  metadata:
+    name: android-sdk
+    labels:
+      app: android-sdk
+      template: android-sdk
+  spec:
+    ports:
+    - name: android-sdk
+      port: 3000
+      protocol: TCP
+      targetPort: 3000
+    selector:
+      run: android-sdk
+    sessionAffinity: None
+    type: ClusterIP
+parameters:
+  - name: LICENSE_AGREEMENT
+    description: You need to enter YES in order to continue.
+    from: "YES"
```

Comment:
I think the YES does not work, in the `from` 

But, it should be possible in the following step, to read the value, and do a manual comparison - in the real APB :smiley_cat: